tests/test_enterprise/test_utils.py

```diff
@@ -22,12 +24,19 @@
     get_default_invite_key_expiration_date,
     get_idiff_list,
     get_platform_logo_url,
+    get_user_details,
     is_pending_user,
     localized_utcnow,
     parse_lms_api_datetime,
     serialize_notification_content,
     truncate_string,
 )
+
+# Fix circular import: ensure we import from project root test_utils, not this file
+project_root = os.path.join(os.path.dirname(__file__), '..', '..')
+if project_root not in sys.path:
+    sys.path.insert(0, project_root)
+
```

Comment:
To fix circular import errors, we mostly directly import it within the test/method.